blueprints/app/files/package.json

```diff
@@ -40,7 +40,8 @@
     "@ember-data/store": "~5.4.0-beta.12",
     "@ember-data/tracking": "~5.4.0-beta.12<% } %>",
     "@ember/optional-features": "^2.2.0",
-    "@ember/test-helpers": "^4.0.4<% if (embroider) { %>",
+    "@ember/test-helpers": "^4.0.4",
+    "@embroider/macros": "^1.16.10<% if (embroider) { %>",
```

Comment:
Needs to come after `@embroider/core`, otherwise, users might end up with changes when their package manager sorts `package.json`.

Reply:
Do we have a tool to do that for us in tests?

Otherwise i can add a test for all the blueprint generation stuff